otterdog/eclipse-pass.jsonnet

```diff
@@ -300,6 +300,15 @@ orgs.newOrg('eclipse-pass') {
       },
     },
     orgs.newRepo('pass-documentation') {
+      allow_merge_commit: true,
+      allow_update_branch: false,
+      delete_branch_on_merge: false,
+      description: "PASS Documentation",
+      dependabot_alerts_enabled: true,
+      dependabot_security_updates_enabled: true,
+      secret_scanning: "disabled",
+      secret_scanning_push_protection: "disabled",
```

Comment:
why are you disabling secret scanning? It might be just a documentation repo, but the checks would still be useful imho.

Reply:
I copied these settings from our `pass-core` repository. If it's best that those settings are set to enabled that works too. I'm not too familiar with all our repository configurations. I can find out for you from our team why it was configured this way.

Reply:
I am pretty sure there is no specific reason it is configured like that. Secret scanning is a quite recent feature in GitHub and most older repos have it disabled ofc. We should really enable it for all repos as its some additional safety measure for free. This can be done in a different PR though.

Reply:
So we're performing a release today. We would like to enable the scanning on all active repos so I will submit a new PR for that in the next couple of days.